spec/datetimeformat.html

```diff
@@ -203,7 +211,10 @@ <h1>BasicFormatMatcher ( _options_, _formats_ )</h1>
             1. If _optionsProp_ is *undefined* and _formatProp_ is not *undefined*, then decrease _score_ by _additionPenalty_.
             1. Else if _optionsProp_ is not *undefined* and _formatProp_ is *undefined*, then decrease _score_ by _removalPenalty_.
             1. Else if _optionsProp_ ≠ _formatProp_,
-              1. Let _values_ be &laquo; `"2-digit"`, `"numeric"`, `"narrow"`, `"short"`, `"long"` &raquo;.
+              1. If _property_ is `"fractionalSecondDigits"`, then
+                1. Let _values_ be &laquo; `0`, `1`, `2`, `3` &raquo;.
```

Comment:
The value `0` should be handled above to ensure `additionPenalty` and `removalPenalty` are properly computed for `fractionalSecondDigits`.

Reply:
addressed in https://github.com/tc39/ecma402/commit/4b135496b8ba91217954e42459ab674daf347fda

Reply:
could you take a look to see is that good enough?